src/ts/waterfall/row/svg-row-subcomponents.ts

```diff
@@ -37,7 +37,7 @@ function makeBlock(rectData: RectData, className: string) {
         foreignElLazy = (getParentByClassName(rect, "water-fall-chart") as Element)
           .querySelector(".tooltip") as SVGForeignObjectElement;
       }
-      showDelayTimeOut = setTimeout(() => {
+      showDelayTimeOut = window.setTimeout(() => {
```

Comment:
is this needed?

Reply:
I can change that sorry, it didn't work without for me it shouldn't be in this commit.